lrathoni/Lullaby-for-a-fish
# Check menus functionnalities and objects textures in a Scene

## What
Added functional Main Menu & Pause Menu
Added 3D models : 3 objects for the game

## How to test
In scene, you will find Main Menu and testLevel, Open them in the Hierarchy.
- Unload testLevel scene
- Load MainMenu scene (if it is necessary)
- Let's play. Select Start demo, result : testLevel scene is loaded
-Press Escape, result : Pause menu appears, you can resume to the game or go back to the main menu by selecting them

In Resource : there are 3 folders for 3 objects
-Check if each texture appear by adding them in the Hierarchy or just look at them in the Project folders
